pkg/api/types.go

```diff
@@ -403,6 +404,13 @@ const (
 	PodUnknown PodPhase = "Unknown"
 )
 
+type PodCondition string
```

Comment:
We use to call Phase Condition, but we changed it (I forget why).  Do we want to use the same name, or something less vague?

If this is the InServiceStatus, could we call it that?